### PR TITLE
Feature/526 crud job template UI

### PR DESCRIPTION
**Pull request by jozefbakus**
Implements: #526

Changes:
- Create, update, delete job template
- List of job template history entries
- Job template history comparison

### CODE REVIEW

File: ui/src/app/components/admin/job-templates/job-template-history/job-template-history.component.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
```
Comment (kevinwallimann) [inline ui/src/app/components/admin/job-templates/job-template-history/job-template-history.component.ts:1]: If we want to add e.g. diffs for the history comparison, or add another crud entity, then we need to refactor this to some abstract component

File: ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          [options]="jobTypesMap"
+          [isRequired]="true">
+        </app-select-part>
+      </div>
+    </div>
+  </div>
+  <div class="card">
+    <div class="card-block">
+      <div class="card-title notSelectable" (click)="toggleJobTemplateParametersAccordion()">
+        <clr-icon shape="angle" [dir]="isJobTemplateParametersHidden?'right':'down'"></clr-icon>
```
Comment (kevinwallimann) [inline ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.html:97]: Amazing. This works without spaces around `?`?

File: ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/*
+/*!
```
Comment (kevinwallimann) [inline ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.scss]: What's this?

File: ui/src/app/components/admin/job-templates/job-template-history/job-template-history.component.ts (hunk 1 above)
Comment (jozefbakus) [reply ui/src/app/components/admin/job-templates/job-template-history/job-template-history.component.ts:1]: I do agree, we will discuss this in another grooming session

File: ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.html (hunk 2 above)
Comment (jozefbakus) [reply ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.html:97]: Yes and it is used also in workflow form

File: ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.scss (hunk 3 above)
Comment (jozefbakus) [reply ui/src/app/components/admin/job-templates/job-templates-form/job-templates-form.component.scss]: Oh, removed

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**